stage2/03-install-umbrel/01-run.sh

```diff
@@ -22,36 +22,44 @@ EOF
 # Install Umbrel
 echo "Installing Umbrel..."
 echo
+
+# Download Umbrel
+mkdir /umbrel
+cd /umbrel
 if [ -z ${UMBREL_REPO} ]; then
-on_chroot << EOF
-mkdir /home/${FIRST_USER_NAME}/umbrel
-cd /home/${FIRST_USER_NAME}/umbrel
 curl -L https://github.com/getumbrel/umbrel/archive/v${UMBREL_VERSION}.tar.gz | tar -xz --strip-components=1
-chown -R ${FIRST_USER_NAME}:${FIRST_USER_NAME} /home/${FIRST_USER_NAME}
-EOF
 else
-on_chroot << EOF
-mkdir /home/${FIRST_USER_NAME}/umbrel
-cd /home/${FIRST_USER_NAME}/umbrel
 git clone ${UMBREL_REPO} -b "${UMBREL_BRANCH}" .
-chown -R ${FIRST_USER_NAME}:${FIRST_USER_NAME} /home/${FIRST_USER_NAME}
-EOF
 fi
 
+# Enable Umbrel OS systemd services
+cd scripts/umbrel-os/services
+UMBREL_SYSTEMD_SERVICES=$(ls *.service)
+echo "Enabling Umbrel systemd services: ${UMBREL_SYSTEMD_SERVICES}"
+for service in $UMBREL_SYSTEMD_SERVICES; do
+    sed -i -e "s/\/home\/umbrel/\/home\/${FIRST_USER_NAME}/g" "${service}"
+    install -m 644 "${service}"   "${ROOTFS_DIR}/etc/systemd/system/${service}"
+    on_chroot << EOF
+systemctl enable "${service}"
+EOF
+done
+
+# Replace /home/umbrel with home/$FIRST_USER_NAME in other scripts
+sed -i -e "s/\/home\/umbrel/\/home\/${FIRST_USER_NAME}/g" "/umbrel/scripts/umbrel-os/umbrel-details"
+
+# Copy Umbrel to image
+mkdir "${ROOTFS_DIR}/home/${FIRST_USER_NAME}/umbrel"
+rsync -qavPHSX /umbrel "${ROOTFS_DIR}/home/${FIRST_USER_NAME}/"
```

Comment:
Would be good if we could comment what these `rsync` options mean.

Alternatively, if they have long alternatives e.g `-q` can be `--quiet`, we could use those instead as they're self documenting.

Reply:
Good idea, will do